src/commands/heroku.rs

```diff
@@ -322,3 +455,50 @@ fn heroku_client(ctx: &Context) -> std::sync::Arc<heroku_rs::framework::HttpApiC
         .expect("Expected Heroku Client Key")
         .clone()
 }
+
+fn heroku_app_config_vars(ctx: &Context, app_name: &str) -> HashMap<String, Option<String>> {
+    let config_var_list = heroku_client(ctx)
+        .request(&config_vars::AppConfigVarDetails { app_id: &app_name })
+        .unwrap();
+    config_var_list
+}
+
+fn block_ips_value(config_vars: HashMap<String, Option<String>>) -> String {
+    config_vars
+        .get(&"BLOCKED_IPS".to_string())
+        .unwrap()
```

Comment:
Pretty sure you're panicking here if the environment variable is missing. This could happen as you call this in `unblock_ip` without verifying beforehand that the variable exists.

Reply:
Good point - adding in handling for that.

Reply:
Done!